api/user/v1/user.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package api.user.v1;
+
+option go_package = "demo/api/user/v1;v1";
+option java_multiple_files = true;
+option java_package = "api.user.v1";
+
+service User {
+	rpc CreateUser (CreateUserRequest) returns (CreateUserReply);
+	rpc UpdateUser (UpdateUserRequest) returns (UpdateUserReply);
+	rpc DeleteUser (DeleteUserRequest) returns (DeleteUserReply);
+	rpc GetUser (GetUserRequest) returns (GetUserReply);
+	rpc ListUser (ListUserRequest) returns (ListUserReply);
+}
+message UserDto {
+	int32 id = 6;
```

Comment:
At some places like these, prefer to use unsigned integers.

Reply:
any logical reasoning behind this, this blog says opposite- https://stackoverflow.com/questions/63105394/in-go-when-should-you-use-uint-vs-int